src/Convert/Unique.hs

```diff
@@ -21,6 +21,23 @@ convert =
 convertStmt :: Stmt -> Stmt
```

Comment:
Minor: At the top of this file, the description of the conversion is now out of date.
Minor: The changelog could be updated, too, but I can take care of this later.